### PR TITLE
Road To Savings

### PR DESCRIPTION
**Pull request by DominicS99**

### CODE REVIEW

File: Week 1/Contest (Mar 11)/Road To Savings/dominic.cpp [hunk 1]
```diff
@@ -0,0 +1,68 @@
+/*
+  Solved after contest
+  
+  Note: Using LL defensively, integer should be able
```
Comment (donaldong) [inline Week 1/Contest (Mar 11)/Road To Savings/dominic.cpp:4]: Yeah, there's no need to use LL here.

File: Week 1/Contest (Mar 11)/Road To Savings/dominic.cpp [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    
+    for (auto &[j, w] : graph[i]) {
+      if (dist[j] > dist[i]+w) {
+        dist[j] = dist[i]+w;
+        pq.emplace(dist[j], j);
+      }
+    }
+  }
+
+  // Backtrace using min dest info
```
Comment (donaldong) [inline Week 1/Contest (Mar 11)/Road To Savings/dominic.cpp:50]: For back tracing, we only need to have a normal queue. PQ  has more overhead than a normal queue

**Review by donaldong — APPROVED** [review]
Neat!

File: Week 1/Contest (Mar 11)/Road To Savings/dominic.cpp (hunk 2 above)
Comment (DominicS99) [reply Week 1/Contest (Mar 11)/Road To Savings/dominic.cpp:50]: Updated!

**Pull request merged**